bedrock/mozorg/templates/mozorg/home/includes/m24/hero.html

```diff
@@ -5,15 +5,60 @@
 #}
 
 <div class="m24-t-light-alt m24-t-dark">
-  <section class="m24-c-flag">
-    <div class="m24-c-flag-media">
-      <img src="{{ static('img/home/2024/hero-symbol-white.svg') }}" alt="" width="216" height="243">
-    </div>
+  <section class="m24-c-flag" data-animation-running="false">
+    <button class="m24-c-flag-button js-animation-button" aria-live="assertive">
```

Comment:
went for assertive here as the text change might be confusing if not announced immediately after button click